go/vt/vtctl/vtctl.go

```diff
@@ -116,6 +116,7 @@ import (
 	"vitess.io/vitess/go/vt/wrangler"
 
 	replicationdatapb "vitess.io/vitess/go/vt/proto/replicationdata"
+	tabletmanagerdatapb "vitess.io/vitess/go/vt/proto/tabletmanagerdata"
```

Comment:
This line is unused (it was, before you could pass `TableSizesOnly` directly to the rpc), and probably will cause `go build` to not succeed

Reply:
already go it but not before you saw :shamebell: